src/models/Room.ts

```diff
@@ -0,0 +1,8 @@
+import { CardScheme } from "./CardScheme";
+
+export interface Room {
+    id : any,
```

Comment:
Is dit echt een `any`? Volgens mij wordt er een UUID v1 in gezet, in welk geval het een `string` is.